### DISCUSSION TITLE
Add keyboard shortcuts/command to the Custom brush in Labs

### DISCUSSION THREAD

**Discussion by sarimarton** [community/community#48554] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hey,

I find myself using the Custom brush prompt more and more times to refactor small chunks which copilot suggested into a different style (like object map to Map, or 'for (let i = 0...' to 'for of' etc). The tool is wonderful. Unfortunately the UI is not too ergonomic for this type of usage, because mousing to the "Custom brush" button is pretty cumbersome to get to the prompt.

Could you add a keyboard shortcuts/command to the Custom brush in Labs so it would be much more ergonomic?

Thanks

**Comment by code-instable** [comment]
(+1)  for custom commands arguments for keyboard.json
